designer/server/src/common/templates/layouts/page.njk

```diff
@@ -12,7 +12,9 @@
   {% include "partials/navigation.njk" %}
 {% endblock %}
 
-{% block pageTitle %}{{ pageTitle }} - {{ config.serviceName }}{% endblock %}
+{% block pageTitle -%}
+  {{ "Error: " if errorList | length }}{{ pageTitle }} - {{ config.serviceName }}
+{%- endblock %}
```

Comment:
I've lifted my suggestion from https://github.com/DEFRA/forms-designer/pull/145#discussion_r1595533400

This prevents the `pageTitle` being modified so we can still use it in question headings

i.e. The prefix `"Error:"` only affects the `<title>` text